contribs/gnodev/pkg/dev/node.go

```diff
@@ -438,3 +391,28 @@ func buildNode(logger *slog.Logger, emitter emitter.Emitter, cfg *gnoland.InMemo
 
 	return node, nil
 }
+
+func newNodeConfig(tmc *tmcfg.Config, appstate gnoland.GnoGenesisState) *gnoland.InMemoryNodeConfig {
+	// Create Mocked Identity
+	pv := gnoland.NewMockedPrivValidator()
+	genesis := gnoland.NewDefaultGenesisConfig(pv.GetPubKey(), tmc.ChainID())
+	genesis.AppState = appstate
```

Comment:
you could add an option to set the chain ID by setting `genesis.ChainID` here

Reply:
How did I miss that? 🤦